### PR TITLE
fix controlled string input

### PR DESCRIPTION
**Pull request by SYBIOTE**
controlled string input did not have appropiate props, they have been fixed and now its workign properly and propagating changes up the chain properly

### CODE REVIEW

**Review by DanielBelmes — APPROVED** [review]

File: packages/editor/src/components/inputs/StringInput.tsx [hunk 1]
```diff
@@ -33,16 +33,27 @@ const inputStyle = {
   margin: 0
 }
 
-interface StyledNumericInputProps {
+interface StyledStringInputProps {
   className?: string
-  onChange?: any
+  onChange?: (e: any) => void
```
Comment (speigg) [inline packages/editor/src/components/inputs/StringInput.tsx:38]: Is there a reason this needs to be `any`? I'd suggest we stick to stricter types.

**Review by speigg — CHANGES_REQUESTED** [review]

File: packages/editor/src/components/inputs/StringInput.tsx (hunk 1 above)
Comment (SYBIOTE) [reply packages/editor/src/components/inputs/StringInput.tsx:38]: the e value?

Comment (speigg) [reply packages/editor/src/components/inputs/StringInput.tsx:38]: yes

**Review by HexaField — APPROVED** [review]

File: packages/editor/src/components/inputs/StringInput.tsx (hunk 1 above)
Comment (SYBIOTE) [reply packages/editor/src/components/inputs/StringInput.tsx]: fixed

**Comment by SYBIOTE** [comment]
@speigg , there is an issue, if we make it strictly typed, there are a lot of higher-level components using the props assuming they were any and they have been interpreted accordingly, do we do ahead and change all the higher-level components as well to fit the stricter type or we change the lower-level string input component with functions to convert the strict react type to string

**Comment by SYBIOTE** [comment]
the main issue is the props being used as base for other props

**Comment by HexaField** [comment]
I belive you functionality you want is what already exists in `StringInput`. `ControlledStringInput` is specific for fields that have dramatic side effects for any value changes at all (such as the ModelComponent loading a scene).

**Pull request closed without merging**